core-skills/frontend-web-development/rhino-assessee.md

```diff
@@ -0,0 +1,41 @@
+# Rhino
+
+This assessment is composed of 3 discrete parts.
+
+## Semantic HTML
+
+
+We have [prepared a HTML page][rhino-semantic-html-page],
+and you will need to change the HTML for more semantic version.
+
+Replace the existing tags with 5 more semantic ones.
+For each change, explain what meaning you thought the original HTML was to convey, and why you chose the semantic tag.
+
+[rhino-semantic-html-page]: https://codepen.io/kjdchapman/pen/zXzQYw
+
+## Basic CSS
+
+You're provided with some [unstyled HTML][rhino-unstyled-html-page],
+which you'll need to add styles such that it looks like the below screenshot.
```

Comment:
Pixel perfect isn't important.

Reply:
Resolved in https://github.com/madetech/learn/pull/119/commits/f92d877f9fe14a423e57ad5c2b9a7d59c78ab9f0